### PR TITLE
user1:adding search user test case

### PR DESCRIPTION
**Pull request by gitcliff**

### CODE REVIEW

File: src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/SearchUserSteps.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    @And("Enter the name of the user in the ‘Search’ text box {string}")
+    public void enterUserNameToSearch(String user) throws Exception {
+
+        manageAccountsPage.clickSearchTextBox(user);
+        Thread.sleep(9000);
+    }
+
+    @Then("the User table should filter, only displaying results of the search")
+    public void resultsOfTheSearch() throws Exception {
+        Thread.sleep(9000);
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/SearchUserSteps.java:65]: I would use waiting elements, forexample waiting for the page to load, something like **pageName.waitForPage:** this waits for 120 milliseconds by default, then **pageName.waitForPageToLoad** waits for 150 secods by default

File: src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/SearchUserSteps.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    @And("Select ‘Manage Accounts’")
+    public void visitManageAccountsAppsPage() throws Exception {
+        manageAccountsPage = systemPage.goToManageAccountsPage();
+    }
+
+    @And("Enter the name of the user in the ‘Search’ text box {string}")
+    public void enterUserNameToSearch(String user) throws Exception {
+        manageAccountsPage.clickSearchTextBox(user);
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/SearchUserSteps.java:55]: This page is not available in the sever we are testing on, for this test to pass, we need to configure this manage accounts Page i the server

**Comment by mozzy11** [comment]
cliff can you document your filter tags on the read me ??

**Comment by mozzy11** [comment]
i also usually add a simple print statement in the [before](https://github.com/IsantePlus/isanteplus-qaframework/blob/main/src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/FindPatientSteps.java#L47) of the steps for easy debugging in the CI environment

File: src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/AddUserSteps.java [hunk 3]
```diff
@@ -36,6 +36,7 @@ public void destroy() {
 
 	@Before(RunTest.HOOK.ADDUSER)
 	public void setLoginPage() {
+		System.out.println(".... User Login......");
```
Comment (mozzy11) [inline src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/AddUserSteps.java]: @gitcliff is this really the User Login Featue??
 please put the corresponding Feature name

**Pull request merged**